server/internal/usecase/interactor/common.go

```diff
@@ -73,3 +74,14 @@ func webhook(ctx context.Context, r *repo.Container, g *gateway.Container, wsID
 	return nil
 
 }
+
+func toEventOperator(o *usecase.Operator) event.Operator {
```

Comment:
It is OK.

```suggestion
func (o *usecase.Operator) EventOperator() event.Operator {
```

at `usecase/operator.go` is better. That idea just hit me.